source/geod24/concurrency.d

```diff
@@ -463,6 +469,11 @@ public class FiberScheduler
         thisScheduler(this);
         scope (exit) thisScheduler(null);
 
+        /// Allow recursively calling dispatch
```

Comment:
What is the use case for this?

Reply:
It was to allow an improvement for the scheduler in the main thread, but thinking about this more, I will take this commit out.